kubernetes/heapster.go

```diff
@@ -88,6 +88,7 @@ func getHeapsterURL(ctx context.Context, clientset kubernetes.Interface, cluster
 	for _, pod := range pods.Items {
 		if strings.Contains(pod.Name, "heapster") {
 			URL.Host = clusterHostURL
+			//nolint staticcheck
```

Comment:
If this is complaining about it being a constant then we should probably just define one for it.

Reply:
This complains about the pod.selfLink saying it is deprecated. I just no linted because this is heapster code that we no longer use.
The linter:
kubernetes/heapster.go:91:15: SA1019: pod.SelfLink is deprecated: selfLink is a legacy read-only field that is no longer populated by the system. +optional (staticcheck)